src/win32_surface.rs

```diff
@@ -86,6 +101,6 @@ impl Deref for Win32Surface {
 
 impl Clone for Win32Surface {
     fn clone(&self) -> Win32Surface {
-        unsafe { from_glib_none(self.to_glib_none().0) }
+        unsafe { Self::from_raw_none(self.to_raw_none()) }
```

Comment:
I can make that change Wednesday (the Windows box is at the office and I am working from home tomorrow).